conda_smithy/tests/test_lint_recipe.py

```diff
@@ -20,9 +20,9 @@ def tmp_directory():
 
 class Test_linter(unittest.TestCase):
     def test_bad_order(self):
-        meta = OrderedDict([['package', []],
-                            ['build', []],
-                            ['source', []]])
+        meta = OrderedDict([['package', {}],
+                            ['build', {}],
+                            ['source', {}]])
```

Comment:
@pelson these are dict-like objects in `ruaml.yaml` and I was getting an error when trying to use `.get` [here](https://github.com/conda-forge/conda-smithy/pull/127/files#diff-75c89ae9a2c5841bf730ed3474cc2987R69). So I made them all dicts. Not sure if that is OK though.